### DISCUSSION TITLE
No answer to instafade suggestion?

### DISCUSSION THREAD

**Discussion by atlant1c** [ppy/osu#20027] [Q&A] [answered]
In #18789, a suggestion for a built in instafade system is made but is locked citing that the question has already been answered in #5913, however the question had not been addressed. The only thing that got a response was the first half of the post discussing a bug with lazers skin system that had caused it to not be possible to instafade, not the suggestion of the feature. Could I get some clarification on the possibility of this feature being added?

**Comment by peppy** [comment, marked as answer]
Are you intending on developing it?

**Reply by atlant1c** [reply]
It appears you did not take kindly to this question so my apologies for asking.
No, I am not, I was simply curious if this feature was on the table at all as a future addition to the game.

**Reply by peppy** [reply]
No further feedback is required then. Supporting "instafade" will come with other skin features which allow more control over animations and style, but these are further away.

We are not going to support this in a hacky way (aka adding a new setting just for it).
